lib/hyrax/controlled_vocabularies/location.rb

```diff
@@ -4,9 +4,9 @@ class Location < ActiveTriples::Resource
       configure rdf_label: ::RDF::Vocab::GEONAMES.name
 
       # Return a tuple of url & label
-      def solrize
+      def solrize(full_label)
```

Comment:
Giving the new parameter a default value of `rdf_label.first` would make this change backwards-compatible:
```suggestion
      def solrize(full_label=rdf_label.first)
```